### PR TITLE
Update welcome dialog

### LINKED ISSUE #24: Update Welcome-Dialog
We have changed the welcome dialog to a nicer tutorial. 

Colors: https://github.com/SecUSo/privacy-friendly-app-example/blob/master/app/src/main/res/values/colors.xml

Java Classes: https://github.com/SecUSo/privacy-friendly-app-example/tree/master/app/src/main/java/org/secuso/privacyfriendlyexample/tutorial

Activity Layout: https://github.com/SecUSo/privacy-friendly-app-example/blob/master/app/src/main/res/layout/activity_tutorial.xml

Tutorial Slides: https://github.com/SecUSo/privacy-friendly-app-example/blob/master/app/src/main/res/layout/tutorial_slide1.xml

### PR DESCRIPTION
**Pull request by Ifill-Brito**
Closes #24

### CODE REVIEW

**Comment by Yonjuni** [comment]
@Ifill-Brito I have reviewed the update. 
Here are a few comments:
1. First Page: 
    - Remove "Welcome to Privacy."
    - Just say 1-2 sentences what the app does.
2. Statistics: Could you rephrase the statistics button to something shorter or move it to an own page?

3. For a new user it is not clear, that the 2nd page refers to lists and the 3rd refers to list items. Maybe add "manage lists"/"manage items" or something similar.

**Comment by Ifill-Brito** [comment]
@Yonjuni I considered your comments in the last push.

1. Welcome message is now shorter
2. "Statistics" tutorial has its own page. I wanted to avoid rewriting the texts because of the translations.
3. I added a title to the tutorial slides as suggested. The Japanese translation for the 2 new titles was done using google translator. I compared DE -> JA, EN -> JA and vice versa to improve the chances that the translation is correct, but it could still be wrong.
